### PR TITLE
Tab.links

### PR DESCRIPTION
**Pull request by josebarata**
[#132239387] Modify tabs & links
+Switched submodule to use JiscMonitor magnificent-octopus

### CODE REVIEW

File: config/service.py [hunk 1]
```diff
@@ -286,6 +286,18 @@
 
 PRIMARY_NAVIGATION = [
     {
+        "label": "About",
+        "url": {
+            "url": "https://monitor.jisc.ac.uk/uk/about",
+        },
+        "visibility": {
+            "auth": True,
+            "anonymous": False
```
Comment (tharglet) [inline config/service.py]: Shouldn't these tabs show all the time and not require auth? This choice seems wrong.

**Review by tharglet — CHANGES_REQUESTED** [review]

**Comment by tharglet** [comment]
@josebarata conflict, also my question is still outstanding?

File: config/service.py (hunk 1 above)
Comment (josebarata) [reply config/service.py]: As added in pivotal tracker: The answer is Yes. However removing or changing the auth property to False doesn't produce any results. Will need further analysis.

**Comment by josebarata** [comment]
Changes were made! The new tabs are now available if the user is not logged in! Ready for another pull review!

**Comment by adam-rehin** [comment]
Changes look ok on paper - I'm assuming the functionality is as required. I note there is a merge conflict to resolve with magnificent-octopus (sub-module?) version.

**Comment by josebarata** [comment]
Resolved. Should I merge it?

**Pull request merged**